### DISCUSSION TITLE
Sidebar keep loading...

### DISCUSSION THREAD

**Discussion by firadhom** [matt8707/ha-fusion#432] [General]
Hi there,

I can't do editing to sidebar. It keeps loading right after I click the edit button; I can't click anything on that view at all because it keeps loading as long as I wait. How to solve this situation?

https://github.com/matt8707/ha-fusion/assets/151456852/64dd7f46-94a2-4c36-ae0a-098b629fccf9

**Comment by Chakkaaa** [comment]
Same here
